04/1_string_case_insensitive_compare/string_utils.c

```diff
@@ -0,0 +1,20 @@
+#include <ctype.h>
+
+#include "string_utils.h"
+
+int string_case_insensitive_compare(const char* str0, const char* str1)
+{
+    int c1 = tolower(*str0);

```

Comment:
from now on.. don't call any function during declaration .. 

```
int c1;
int c2;

c1 = tolower(*str0);
c2 = tolower(*str1);
```
is the way to go